src/Components/Artwork/GridItem.tsx

```diff
@@ -61,46 +58,29 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
     isMounted: false,
   }
 
+  canHover: boolean
+
   componentDidMount() {
     this.setState({
       isMounted: true,
     })
+
+    this.canHover = !window.matchMedia("(hover: none)").matches
   }
 
-  getImageUrl(breakpoints) {
+  getImageUrl() {
     const imageURL = this.props.artwork.image.url
-
     if (!imageURL) {
       return null
     }
 
-    if (this.state.isMounted) {
-      const breakpoint = Object.keys(pickBy(breakpoints))[0]
-      const getWidth = () => {
-        switch (breakpoint) {
-          case "xs":
-          case "sm":
-          case "md":
-            return 400
-          case "lg":
-          case "xl":
-          default:
-            return 300
-        }
-      }
-
-      const width = Math.floor(getWidth())
-      const height = Math.floor(width / this.props.artwork.image.aspect_ratio)
-
-      // Either scale or crop, based on if an aspect ratio is available.
-      const type = this.props.artwork.image.aspect_ratio ? "fit" : "fill"
-      // tslint:disable-next-line:max-line-length
-      return `${
-        sd.GEMINI_CLOUDFRONT_URL
-      }/?resize_to=${type}&width=${width}&height=${height}&quality=${IMAGE_QUALITY}&src=${encodeURIComponent(
-        imageURL
-      )}`
-    }
+    const width = 300
```

Comment:
@mzikherman - Sticking with a concrete value solves a whole host of issues without introducing any apparent degradation.

Reply:
Does this wind up being upscaled when rendered at the `md` breakpoint? (Just looking at the mapping above tha width is 400 vs 300 for the rest, but I'm not sure if that's how wide the columns are). If so, we might want to set this to 400.

Reply:
Updated.